tests/integration-tests/tests/ad_integration/test_ad_integration/test_ad_integration/ad_stack.yaml

```diff
@@ -203,7 +203,7 @@ Resources:
                         --region {{ region }} \
                         --directory-id "\${DIRECTORY_ID}" \
                         --user-name "\${NEW_USER_ALIAS}" \
-                        --new-password "ApplesBananasCherries!"  # TODO: generate random passwords
```

Comment:
unrelated to the PR: i got really good at typing `ApplesBananasCherries!` over the past two weeks